Framework/src/runBasic.cxx

```diff
@@ -133,6 +133,6 @@ std::string getConfigPath(const ConfigContext& config)
   // The the optional one by the user
   auto userConfigPath = config.options().get<std::string>("config-path");
   // Finally build the config path based on the default or the user-base one
-  std::string path = std::string("json:/") + (userConfigPath.empty() ? defaultConfigPath : userConfigPath);
```

Comment:
That didn't work?

Reply:
apparently it lacks a `/`. for sure on Mac it failed. I'll give it a try on CC7.

Reply:
ok, for CC7 both worked as far as I remember

Reply:
I cannot test at the moment due to the alicache problem.

Reply:
ok it works both on Mac and CC7 now

Reply:
ok, i don't mind either of these alternatives, just wanted to know if both work.